lab-jacob/test/greet-test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const greet = require(__dirname + '/../lib/input');
+const assert = require('assert');
+
+describe('testing input', function() {
+  describe('testing the greeting', function () {
+    it ('should assign "jacob" to argument and return hello jacob', function() {
+      process.argv[2] = 'jacob';
+      assert.equal(greet(process.argv[2]), 'hello jacob');
```

Comment:
You are only testing passing a name in as a prarameter in this test.
you are not testing process.argv